### PR TITLE
add argument to pass default encoding to JVM 

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

add argument to pass default encoding to JVM 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: distroless/Dockerfile [hunk 1]
```diff
@@ -28,6 +28,8 @@ ONBUILD ARG DEV_MODE
 ONBUILD ARG DEV_JVM_ARGS=${DEV_MODE:+'-XX:InitialRAMPercentage=20.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=10.0 -XX:+UseParallelOldGC -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Xms128M'}
 ONBUILD ARG JVM_ARGS=${DEV_JVM_ARGS:-'-XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC'}
 
+ONBUILD ARG JVM_ENCODING_ARGS='-Dfile.encoding=UTF-8'
```
Comment (timja) [inline distroless/Dockerfile]: also needs doing in https://github.com/hmcts/cnp-java-base/blob/master/distroless/debug.Dockerfile

Comment (mdayican) [reply distroless/Dockerfile]: added

[the remark this answers is not in the document]
Comment (luigibk) [reply distroless/Dockerfile]: There ... which is what (standard) JAVA_OPTS is for. I can see no need for another arg. It just adds unnecessary complexity imo

**Pull request merged**